asyncapi-cli/src/main/java/io/ballerina/asyncapi/codegenerator/entity/Visitor.java

```diff
@@ -0,0 +1,182 @@
+package io.ballerina.asyncapi.codegenerator.entity;
```

Comment:
Shall we add the license header?

Reply:
Do we need this class?

Reply:
No we didn't need it. Sorry, that was left out by mistake, removed it in https://github.com/ballerina-platform/asyncapi-tools/pull/119/commits/9aa9c56e65b8225569946ccee43f86878bcc4790